src/org/omegat/util/Preferences.java

```diff
@@ -187,6 +187,8 @@
     public static final String ALLOW_YANDEX_TRANSLATE = "allow_yandex_translate";
 
     public static final String ALLOW_MOSES_TRANSLATE = "allow_moses_translate";
+    
+    public static final String ALLOW_MTHUB_TRANSLATE = "allow_MTHUB_translate";
```

Comment:
Sorry for the trivial comment, but the preference string should be all lowercase.

Reply:
When I change it, it doesn't works. I can't find out why. Do I need to change anything else?

Reply:
The only thing I can think of is that the key changed so you need to either edit your `omegat.prefs` file to match (make sure OmegaT is closed when you do so), or you need to set your credentials again in Preferences.